xc7/primitives/slicem/srl/srlc32e_vpr.pb_type.xml

```diff
@@ -6,12 +6,29 @@
   <output name="Q" num_pins="1"/>
   <output name="Q31" num_pins="1"/>
 
-  <T_setup port="CE" clock="CLK" value="1e-11"/>
-  <T_setup port="A" clock="CLK" value="1e-11"/>
-  <T_setup port="D" clock="CLK" value="1e-11"/>
+  <T_setup port="CE" clock="CLK" value="{setup_CLK_WE}"/>
+  <T_hold port="CE" clock="CLK" value="{hold_CLK_WE}"/>
+  <T_setup port="D" clock="CLK" value="{setup_CLK_DI1}"/>
+  <T_hold port="D" clock="CLK" value="{hold_CLK_DI1}"/>
 
-  <T_clock_to_Q port="Q" clock="CLK" max="1e-11"/>
-  <T_clock_to_Q port="Q31" clock="CLK" max="1e-11"/>
+  <delay_matrix type="min" in_port="A" out_port="Q">
+    {iopath_A2_O6}
```

Comment:
please add a comment here explaining why we start from A2

Reply:
Done.